Installation.md

```diff
@@ -66,7 +67,13 @@ haxelib install hlsdl
 haxelib install hldx
 ```
 
+You can also install these libraries directly from github to get the latest versions (**recommended**):
+```
+haxelib git hlopenal https://github.com/HaxeFoundation/hashlink master libs/openal
+haxelib git hlsdl https://github.com/HaxeFoundation/hashlink master libs/sdl
+haxelib git hldx https://github.com/HaxeFoundation/hashlink master libs/directx
+```
 ## Verify the installation
-Once everything is setup, you should be able to run `hl` command from your terminal:
+Once everything is setup, you should be able to run `hl` command from your terminal (note: the version will likely be different):
```

Comment:
Make reader aware that the version in the screenshot and the version that they will have will be likely different.